<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gs_FrequencyDialog</class>
 <widget class="QDialog" name="gs_FrequencyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>变频器参数</string>
  </property>
  <widget class="QPushButton" name="m_btn_ControlFrequency">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_ControlAir">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>540</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_Default">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>默认值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_Sure">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="m_tabwidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>271</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="m_tab_WorkPara">
    <attribute name="title">
     <string>工作参数</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>转速:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_lineED_WorkCircle">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string> 加速度:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_lineED_WorkAcc">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_lineED_WorkDec">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string> 减速度:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="m_combox_WorkDirect">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <item>
      <property name="text">
       <string>正转</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>反转</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>    方向:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="m_tab_JogPara">
    <attribute name="title">
     <string>Jog参数</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>转速:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="m_lineED_JogCircle">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>  方向:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="m_combox_JogDirect">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>正转</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>反转</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="m_btn_readStatus">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>610</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>读取状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>速度:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>频率:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>电流:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_lineED_FrequencySpeed">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_lineED_FrequencyPin">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_lineED_FrequencyCurrent">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>m_tabwidget</zorder>
  <zorder>m_btn_ControlFrequency</zorder>
  <zorder>m_btn_ControlAir</zorder>
  <zorder>m_btn_Default</zorder>
  <zorder>m_btn_Sure</zorder>
  <zorder>m_btn_readStatus</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_9</zorder>
  <zorder>m_lineED_FrequencySpeed</zorder>
  <zorder>m_lineED_FrequencyPin</zorder>
  <zorder>m_lineED_FrequencyCurrent</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
